ibm/mas_devops/roles/suite_install/README.md

```diff
@@ -11,10 +11,10 @@ Role Variables
 - `mas_channel` Defines which channel of MAS to subscribe to
 - `mas_domain` Opitional fact, if not provided the role will use the default cluster subdomain
 - `mas_instance_id` Defines the instance id to be used for MAS installation
-- `mas_icr_cp` Defines the entitled registry from the images should be pulled from. Set this to `cp.icr.io/cp` when installing release version of MAS or `wiotp-docker-local.artifactory.swg-devops.com` for dev
-- `mas_icr_cpopen` Defines the registry for non entitled images, such as operators. Set this to `icr.io/cpopen` when installing release version of MAS or `wiotp-docker-local.artifactory.swg-devops.com` for dev
-- `mas_entitlement_username` Username for entitled registry. This username will be used to create the image pull secret. Set to `cp` when installing release or use your `w3Id` for dev.
-- `mas_entitlement_key` API Key for entitled registry. This password will be used to create the image pull secret. Set to with IBM entitlement key when installing release or use your artifactory `apikey` for dev.
+- `mas_icr_cp` Defines the entitled registry from the images should be pulled from. Set this to `cp.icr.io/cp` when installing release version of MAS or `wiotp-docker-local.artifactory.swg-devops.com` for dev or the local registry endpoint when installing in an air gap environment.
```

Comment:
The user does not need to set this in an airgap environment where image mirroring (and reference by digest) uses the ImageContentSourcePolicy to auto-redirect from the original source to the mirror; it's only used in development to set up Artifactory as an alternative for ICR.